timeshift-autosnap-apt

```diff
@@ -51,7 +51,17 @@ fi
 
 readonly SNAPSHOT_DESCRIPTION=$(get_property "snapshotDescription" "string" "{timeshift-autosnap-apt} {created before upgrade}")
 
-timeshift --create --comments "$SNAPSHOT_DESCRIPTION" || { echo "Unable to run timeshift-autosnap-apt! Please close Timeshift and try again. Script will now exit..." >&2; exit 1; }
+# This is a makeshift do-while loop, as per https://stackoverflow.com/a/16489942
+while true; do
+    SNAPSHOT_SUCCESSFUL=1
+    if ! timeshift --create --comments "$SNAPSHOT_DESCRIPTION"; then
+        echo "Unable to run timeshift-autosnap-apt! Please close Timeshift and try again. Script will exit in 30 seconds." >&2
+        read -t30 -n1 -p"Press any key  to try again. " || exit 1
```

Comment:
If the `timeshift` command fails, we throw an error message and start a 30-second countdown. Now, the user has time to close Timeshift and try taking the snapshot again by pressing any key.
If we recieve no user input within 30 seconds, the script exits by throwing an error.